cpp/cmake/thirdparty/CUDF_GetcuCollections.cmake

```diff
@@ -0,0 +1,38 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+function(find_and_configure_cucollections)
+
+    if(TARGET cuCollections::cuco)
```

Comment:
Currently cuCollections doesn't create a target with the name `cuCollections::cuco` it only creates the target `cuco`, so this checks is incorrect.


The call to `fix_cmake_global_defaults` doesn't need to exist as we aren't using the  `cuCollections::cuco` or `cuco` targets at all.